Source/Core/VideoBackends/D3D/D3DState.cpp

```diff
@@ -299,7 +299,7 @@ ID3D11SamplerState* StateCache::Get(SamplerState state)
   sampdc.AddressV = address_modes[static_cast<u32>(state.wrap_v.Value())];
   sampdc.MaxLOD = state.max_lod / 16.f;
   sampdc.MinLOD = state.min_lod / 16.f;
-  sampdc.MipLODBias = (s32)state.lod_bias / 32.0f;
+  sampdc.MipLODBias = (s32)state.lod_bias / 256.f;
```

Comment:
I noticed 256 is specified in a number of places.  What does it represent?  It may be useful to make it some constant, that way it isn't just a magic number.. (not sure what those 16s are either but that's for another time)

Reply:
So, from what I gathered lod_bias was an 8 bit int representing a float with 8 integers, each subdivided in 32 floats, so the smallest delta bias you could represent was 0.03 and the max bias was 8.
That turns out, wasn't neither precise or wide enough for this, so what I did was to increase the bias to 16 bit. This way it represents a bias with steps of 0.004 and a max of 256. 
Tbh 14 or 13 bits (64 or 32 max) would probably have been enough, but I didn't see a reason to spare bits.. I had to make `SamplerState` an `u64` either way.

As far as I could see, all the magic numbers come from the GC's architecture itself, because `BPMemory.h` hardcodes them as 8 bit. So probably that's just how it worked in the hardware.
And yeah, the only reason to keep that encoding in `SamplerState` seems to be able to fit it in an `u32` so they're easier to lookup.

tl;dr: 256 is just the precision.

Reply:
I agree with "And yeah, the only reason to keep that encoding in SamplerState seems to be able to fit it in an u32 so they're easier to lookup.", but as we don't fit in u32 any more, we can use any precision we want, including a 32bit float. The later might be easier for the backends ;)

Reply:
If we use 3 floats it becomes more than u64 though... and if we only make bias a float, it's kinda asymmetric? I think it's it's not all that bad as is, actually, but you tell me!